span-normalizer/span-normalizer/build.gradle.kts

```
@@ -48,8 +48,12 @@ dependencies {
 
   // Required for the GRPC clients.
   runtimeOnly("io.grpc:grpc-netty:1.35.0")
-  runtimeOnly("io.netty:netty-codec-http2:4.1.59.Final")
-  runtimeOnly("io.netty:netty-handler-proxy:4.1.59.Final")
+  runtimeOnly("io.netty:netty-codec-http2:4.1.59.Final") {
+    because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439")
+  }
+  runtimeOnly("io.netty:netty-handler-proxy:4.1.59.Final") {
+    because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439")
+  }
```

```suggestion
constraints {
  runtimeOnly("io.netty:netty-codec-http2:4.1.59.Final") {
    because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439")
  }
  runtimeOnly("io.netty:netty-handler-proxy:4.1.59.Final") {
    because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439")
  }
}
```

(probably want to move them out of line with the other dependencies too)

---

The difference of using a constraint here is that it doesn't actually declare a direct dependency, it just constrains the version if it's otherwise present.